releasenotes/notes/remove-deprecated-items-2.0-89eac06472c59554.yaml

```
@@ -89,4 +89,22 @@ upgrade:
     ``ddtrace.filters.TraceCiVisibilityFilter`` is removed.
 
 
+    "
+  - ".. _remove-dd-log-format:
+
+
+    ``ddtrace.tracer.DD_LOG_FORMAT`` is removed.
+    As an alternative, please follow the log injection formatting as provided in the
+    `log injection docs <https://ddtrace.readthedocs.io/en/stable/advanced_usage.html#update-log-format>`_.
+
+
+    "
+  - ".. _remove-basic-config:
+
+
+    ``DD_CALL_BASIC_CONFIG`` is removed.
+    There is no special configuration necessary to replace ``DD_CALL_BASIC_CONFIG``.
+    The ddtrace logger will log to standard out by default or to a file specified by ``DD_TRACE_LOG_FILE``.
```

```suggestion
    The ddtrace logger will log to standard out by default or additionally to a file specified by ``DD_TRACE_LOG_FILE``.
```

? this is a nit pick, not sure this really adds much clarification